src/screens/AirdropDashboard/components/AirdropInProgressContent.tsx

```diff
@@ -33,12 +33,33 @@ export const AirdropInProgressContent: FC<Props> = ({
   usersQuantity,
 }) => {
   const [communityCarouselActive, setCommunityCarouselActive] = useState(false);
+  const [loadingWalletsIds, setLoadingWalletsIds] = useState<string[]>([]);
   let _carouselRef: Carousel<AirdropCarouselCardData>;
 
   const setRef = (carouselRef: Carousel<AirdropCarouselCardData>) => {
     _carouselRef = carouselRef;
   };
 
+  useEffect(() => {
+    const idsToRemove: string[] = [];
+
+    loadingWalletsIds.forEach(id => {
+      if (!availableWallets.filter(w => w.id === id)[0]) {
+        idsToRemove.push(id);
+      }
+    });
+
+    const stillLoading = loadingWalletsIds.filter(id => !idsToRemove.includes(id));
+
+    setLoadingWalletsIds(stillLoading);
+  }, [availableWallets, loading]);
+
+  const _subscribeWallet = (wallet: Wallet) => {
```

Comment:
Is it a private method of function?

Reply:
just a wrapper for prop function. Added _ as I didnt have better name in mind and yes, it is private :)